### DISCUSSION TITLE
如果全力拯救，以国企、城投等方式注资，大概需要多少，这些钱会对我们产生什么影响

### DISCUSSION THREAD

**Discussion by vvspWsvzz1z7** [WeNeedHome/SummaryOfLoanSuspension#972] [General]
比如在二线城市五千的工资还能够住城中村和吃饱饭吗？

**Comment by volans19** [comment]
这个如果基本是0可能，但是万万万一真有这个如果，无论需要多少，99.9%都会被一层层的青天大老爷小老爷吃掉，然后摆摆样子，等一个礼拜，最多俩礼拜，老百姓再次选择忘记，产生什么影响，结论：
0

**Comment by TALK1949** [comment]
开玩笑，地方政府是靠着房地产赚钱，现在没钱赚了还让他掏钱，那是根本不可能地，不然为啥现在强迫让公务员买房，自己养自己。
国企就更别提了，国企不趁着这时候打劫这些房地产商的优质资产就不错了。

**Comment by Accelerator6666** [comment]
你这想让那些官老爷公务员把吃下去的全吐出来？现实吗？

**Comment by ddb4github** [comment]
一个小区从停工恢复到正常使用需要一千万，城投、国企、银行、房管、银监过一遍水，花了两千万。房屋交付后，水暖电气接入、水暖电气物业集体涨价，房地产税，最后回收四千万，其中一千五百万填补之前银行贷出的缺口，五百万和利息做坏账处理，报央妈核销

**Comment by cc2218** [comment]
那就是印钱。100W烂尾人的痛苦给全国13.85亿人分摊。
但是真正犯罪的那些人赚更多。

**Reply by vvspWsvzz1z7** [reply]
> 那就是印钱。100W烂尾人的痛苦给全国13.85亿人分摊。 但是真正犯罪的那些人赚更多。

不印钱能咋办，等传导到所有人身上吗
